pylint/checkers/typecheck.py

```diff
@@ -1831,12 +1830,10 @@ def _check_unsupported_alternative_union_syntax(self, node: nodes.BinOp) -> None
                 self.add_message("unsupported-binary-operation", args=msg, node=node)
                 break
 
-    @check_messages("unsupported-binary-operation")
     def _visit_binop(self, node: nodes.BinOp) -> None:
```

Comment:
Do you understand why these are underscored? They don't seem to be called in this file..

Reply:
I saw that the functional test for this message (``unsupported-binary-operation``) is always skipped, with the following comment:
```
Disable for unsupported-binary-operation

Unfortunately, this warning exhibits currently way too many false
positives, practically rendering this error useless. I can't fix
all of them until I release Pylint 1.5, so the most reasonable
choice for now is to disable it and reenable it as soon as we fix
those problems.
```

Reply:
Oh wow, we should probably follow up on that..

But then the decorator is "correct" we only need to de-underscore this function right? Perhaps we should keep the decorator and add a TODO to fix this?

Reply:
You are right - if we plan to re-enable this check, the decorator should stay. Will fix this and add a TODO.